stacks_queues/stack.py

```diff
@@ -21,13 +24,22 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        
+        if self.store.length() == 0:
+            return None
+        else:
+            return self.store.remove_first()
```

Comment:
Per the specification, you don't want to return anything!
```suggestion
            self.store.remove_first()
```